Framework/API/inc/MantidAPI/FunctionValues.h

```diff
@@ -101,6 +101,9 @@ class MANTID_API_DLL FunctionValues {
   /// Return the calculated values as a vector
   std::vector<double> toVector() const { return m_calculated; }
 
+  /// Return a reference to the calculated values
+  std::vector<double> &getCalculated();
```

Comment:
If this method isn't used I would prefer to remove it. Non-virtual methods getCalculated(size_t) or operator[](size_t) can be used to access the calculated data.